ui/main/src/app/modules/usercard/usercard.component.ts

```diff
@@ -109,6 +109,8 @@ export class UserCardComponent implements OnDestroy, OnInit {
     endDateVisible = true ;
     lttdVisible = true ;
     recipientVisible = true; // if recipientVisible == false, then selectedRecipients = recipientList
+    entitiesAllowedToRespond;
+    entitiesRequiredToRespond;
```

Comment:
It seems you do not used these variables ?

Reply:
yes, they are used below, in lines 555-560

Reply:
in 555-560 it s local variables so not these ones

Reply:
fixed